tests/shard_map_test.py

```diff
@@ -2162,6 +2164,48 @@ def f():
 
     self.assertAllClose(f(), np.array(range(4), dtype=np.int32).reshape(-1, 1))
 
+  def test_partial_auto_ppermute(self):
+    if xla_extension_version < 302:
+      self.skipTest('minimum xla extension version 302')
+    if config.use_shardy_partitioner.value:
+      self.skipTest('Shardy does not support full-to-shard.')
+
+    mesh = jtu.create_mesh((4, 2), ('i', 'j'))
+    x = jnp.arange(8.)
+
+    def g(x):
+      x = jax.lax.with_sharding_constraint(x, NamedSharding(mesh, P('j')))
```

Comment:
If we add a line here,

`jax.debug.print("x={x}", x=x)

It will crash. Since Parker already has a fix in XLA, maybe it is convenient to include this test as well.
Thanks.

Reply:
Is there an additional bug with printing?

The intention is that this test would crash until https://github.com/openxla/xla/pull/20656 gets submitted (and we rebuild jaxlib), so we skip it based on a version check. (There's one CL that is generating both PRs.)